src/components/Common/SearchInput.tsx

```diff
@@ -365,15 +375,15 @@ export default function SearchInput({
                             return (
                               <CommandItem
                                 key={option.key}
-                                onSelect={() =>
+                                onSelect={() => {
                                   handleOptionChange(
                                     safeOptions.findIndex(
                                       (option) =>
                                         option.key ===
                                         unselectedOptions[index].key,
                                     ),
-                                  )
-                                }
+                                  );
```

Comment:
shouldn't need to be encapsulated since it's just calling one fn; try removing the semicolon.